geopandas/array.py

```diff
@@ -1044,25 +1044,30 @@ def fillna(self, value=None, method=None, limit=None):
             Alternatively, an GeometryArray 'value' can be given. It's expected
             that the GeometryArray has the same length as 'self'.
 
-        method : {'backfill', 'bfill', 'pad', 'ffill', None}, default None
-            Method to use for filling holes in reindexed Series
-            pad / ffill: propagate last valid observation forward to next valid
-            backfill / bfill: use NEXT valid observation to fill gap
+        method : {'backfill', 'bfill', 'ffill', None}, default None
+            Method to use for filling holes in reindexed GeoSeries:
+
+            - ffill: propagate last valid observation forward to next valid.
```

Comment:
based on pandas 2.0

Reply:
Pad is an alias for `ffill` so I think it's fine to leave out of the docs here then.